src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java

```diff
@@ -536,6 +531,7 @@ WaitingProcessOutputLineListener getRunDevAppServerWaitListener() {
     private List<CloudSdkResolver> resolvers;
     private int runDevAppServerWaitSeconds;
     private boolean inheritProcessOutput;
+    private Path javaBinaryPath = Paths.get(System.getProperty("java.home"));
```

Comment:
perhaps we should just store the java home, since that's what most tools present as the path of interest to the user.